client/components/data/domain-management/index.jsx

```diff
@@ -70,6 +70,7 @@ class DomainManagementData extends Component {
 
 export function UsePresalesChat() {
 	usePresalesChat( 'wpcom', true, true );
+	return null;
```

Comment:
React expects every component to return a valid node. `undefined` or `void` are not valid nodes, whereas `null` is.

Since we don't want to render anything with this component, we need to return `null` here.

If you try to return nothing from a component, React will yell at you 😅

Reply:
Yeah it was an issue already but it hadn't been noticed until I introduced Typescript into the domains area. And Typescript picked up the issue immediately :)